validation-test/Reflection/reflect_UInt16.swift

```diff
@@ -4,7 +4,7 @@
 
 // RUN: %target-run %target-swift-reflection-test %t/reflect_UInt16 | %FileCheck %s --check-prefix=CHECK-%target-ptrsize
 
-// REQUIRES: objc_interop
+// REQUIRES: asserts || os=macosx
```

Comment:
Nice! But I think it should be 'asserts || objc_interop', since we want to test on tvOS/watchOS etc.